src/rx-query.ts

```diff
@@ -747,16 +914,18 @@ export async function queryCollection<RxDocType>(
                 docs.push(docData);
             }
         }
-    } else {
-        const preparedQuery = rxQuery.getPreparedQuery();
-        const queryResult = await collection.storageInstance.query(preparedQuery);
-        if (rxQuery._limitBufferSize !== null && rxQuery.mangoQuery.limit && queryResult.documents.length > rxQuery.mangoQuery.limit) {
-            // If there are more than query.limit results, we pull out our buffer items from the
-            // last rxQuery._limitBufferSize items of the results.
-            rxQuery._limitBufferResults = queryResult.documents.splice(rxQuery.mangoQuery.limit);
-        }
-        docs = queryResult.documents;
+        return docs;
```

Comment:
likewise i know his formatting here sucks, but for the sake of future upgrades to rxdb15, 16, 17, etc can we keep the formatting the same?

Reply:
yeah… i don't mind reverting this to its previous state, merging upstream changes is going to be fun anyway